erica_app/erica/elster_xml/est_mapping.py

```diff
@@ -20,9 +20,9 @@
     'person_a_address_ext': 'E0101301',
     'person_a_plz': 'E0100601',
     'person_a_town': 'E0100602',
-    'person_a_disability_degree': 'E0109708',
-    'person_a_has_merkzeichen_bl_tbl_h_pflegegrad': 'E0109706',
-    'person_a_has_merkzeichen_g_ag': 'E0109707',
+    'person_a_pauschbetrag_disability_degree': 'E0109708',
+    'person_a_pauschbetrag_has_merkzeichen_bl_tbl_h_pflegegrad': 'E0109706',
+    'person_a_pauschbetrag_has_merkzeichen_g_ag': 'E0109707',
 
```

Comment:
Wait a second, aren't we missing person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad etc? Where are they mapped to eric e-numbers?

Reply:
Very good catch!